packages/components/src/styles/components/code-editor/theme.scss

```
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// THEMING
+
+.hds-code-editor--theme-dark {
+  // COLORS
+
+  // Base UI colors:
+  --hds-code-editor-color-border-strong: rgba(178, 182, 189, 40%);
+  --hds-code-editor-color-foreground-primary: #d5d7db;
+  --hds-code-editor-color-foreground-faint: #878a8f;
+  --hds-code-editor-color-foreground-high-contrast: #fff;
+  --hds-code-editor-color-surface-faint: #15181e;
+  --hds-code-editor-color-surface-primary: #0D0E12;
```

Need to add the variable for our button's active state:
```suggestion
  --hds-code-editor-color-surface-primary: #0D0E12;
  --hds-code-editor-color-surface-interactive-active: #2B303C;
```